python/pylibraft/pylibraft/neighbors/ivf_pq/ivf_pq.pyx

```diff
@@ -135,9 +134,11 @@ cdef class IndexParams:
         n_list : int, default = 1024
             The number of clusters used in the coarse quantizer.
         metric : string denoting the metric type, default="l2_expanded"
-            Valid values for metric: ["l2_expanded", "inner_product"], where
-            - l2_expanded is the equclidean distance without the square root
+            Valid values for metric: ["l2_expanded", "inner_product",
```

Comment:
Doesn't need to be done in this PR, but I think `l2_expanded` is equivalent to `sqeuclidean` in this context and we should probably accept that as an option as well (since it's likely to be more familiar to users) .

Reply:
https://github.com/rapidsai/raft/pull/1160